spec/features/trainer_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+describe "training without cards" do
+  before :each do
+    create(:card)
+  end
+
+  it 'no cards' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.